### PR TITLE
Fix typo

### PR DESCRIPTION
**Pull request by maarcingebala**
This PR fixes typo: "accomodation" -> "accommodation".

### Pull Request Checklist

1. [ ] Privileged views and APIs are guarded by proper permission checks.
1. [ ] All visible strings are translated with proper context.
1. [ ] All data-formatting is locale-aware (dates, numbers, and so on).
1. [ ] Database queries are optimized and the number of queries is constant.
1. [ ] The changes are tested.
1. [ ] The code is documented (docstrings, project documentation).
1. [ ] GraphQL schema and type definitions are up to date.

### CODE REVIEW

File: saleor/static/dashboard-next/storybook/__snapshots__/Stories.test.ts.snap [hunk 1]
```diff
@@ -37876,7 +37876,7 @@ exports[`Storyshots Views / Products / Product list default 1`] = `
             <td
               class="MuiTableCell-root-id MuiTableCell-body-id Component-textRight-id"
             >
-              XAG 274.99
+              XAG274.99
```
Comment (patrys) [inline saleor/static/dashboard-next/storybook/__snapshots__/Stories.test.ts.snap]: Hmm, that's unexpected.

Comment (maarcingebala) [reply saleor/static/dashboard-next/storybook/__snapshots__/Stories.test.ts.snap]: I just did `npm test -- -u` as @dominik-zeglen suggested.

Comment (patrys) [reply saleor/static/dashboard-next/storybook/__snapshots__/Stories.test.ts.snap]: @dominik-zeglen Can you look into this?

Comment (dominik-zeglen) [reply saleor/static/dashboard-next/storybook/__snapshots__/Stories.test.ts.snap]: Old node version

**Pull request merged**